include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -135,14 +135,13 @@ class FeatureModelModification {
 
 protected:
   /// \brief Set the parrent of a Feature.
-  static void setParent(Feature &F, Feature *Parent) {
-    F.setParent(Parent);
-  }
+  static void setParent(Feature &F, Feature *Parent) { F.setParent(Parent); }
 
-  static void addChild(Feature &F, Feature &Child) {
+  static void addChild(Feature &F, Feature *Child) {
```

Comment:
```suggestion
  static void addChild(Feature &F, Feature &Child) {
```
There is no go way to handle nullptrs here, so why should `Child` be a ptr?

Reply:
Good point as we talked about it last week -- there is no reason, as any `nullptr` should be handled beforehand -- will revert that an update references accordingly.